src/components/Dropdowns/StyledSelect.tsx

```diff
@@ -34,6 +34,7 @@ const LABELED_COLOR_INDICATOR_BASE_STYLE = {
   marginRight: 6,
   height: 12,
   width: 12,
+  minWidth: 12,
```

Comment:
Fixes a bug I noticed where at narrow aspect ratios, the annotation type indicator would get squished. 

This change keeps the square styling: 
![image](https://github.com/user-attachments/assets/dcebe563-7b26-45eb-bc1e-2dbd20a374a3)